### PR TITLE
[DEVOPS-978] Add option to demo cluster to disable client auth

### PR DESCRIPTION
**Pull request by disassembler**
## Description

Adds an attribute `demoClusterDaedalusDev` to `default.nix` that frontend testers can use to launch a demo cluster with daedalus.

## Linked issue

https://iohk.myjetbrains.com/youtrack/issue/DEVOPS-978

## Type of change

- [ ] 🐞 Bug fix (non-breaking change which fixes an issue)
- [ ] 🛠 New feature (non-breaking change which adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🏭 Refactoring that does not change existing functionality but does improve things like code readability, structure etc
- [ ] 🔨 New or improved tests for existing code
- [ ] ⛑ git-flow chore (backport, hotfix, etc)

## Developer checklist

- [ ] I have read the [style guide](https://github.com/input-output-hk/cardano-sl/blob/develop/docs/style-guide.md) document, and my code follows the code style of this project.
- [ ] If my code deals with exceptions, it follows the [guidelines](https://github.com/input-output-hk/cardano-sl/blob/develop/docs/exceptions.md).
- [ ] I have updated any documentation accordingly, if needed. Documentation changes can be reflected in opening a PR on [cardanodocs.com](https://github.com/input-output-hk/cardanodocs.com), amending the inline [Haddock](https://www.haskell.org/haddock/) comments, any relevant README file or one of the document listed in the [docs](https://github.com/input-output-hk/cardano-sl/tree/develop/docs) directory.

## Testing checklist

- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: default.nix [hunk 1]
```diff
@@ -20,6 +20,7 @@ in
 , enableDebugging ? false
 , enableBenchmarks ? true
 , allowCustomConfig ? true
+, disableClientAuth ? true
```
Comment (cleverca22) [inline default.nix]: nothing appears to be reading this new config flag

**Comment by darko-mijic** [comment]
@disassembler, please provide instructions how to:
- build
- start demo cluster with TLS
- start demo cluster without TLS

**Comment by darko-mijic** [comment]
@disassembler, we also need to update the documentation.

**Comment by disassembler** [comment]
I've renamed `demoClusterDisableAuth` to `demoClusterDaedalusDev`. It also no longer imports the 11 genesis keys when using the Daedalus Dev option.

**Pull request merged**